README.md

```diff
@@ -1971,4 +1971,244 @@ You can create a `./gradlew checkstyle` command using the following:
 task checkstyle {
     dependsOn(checkstyleMain, checkstyleTest)
 }
-```
\ No newline at end of file
+```
+
+## TDD with Fibonacci
+
+As with all Test Driven Development, the key things to remember are:
+
+1. Red: Write as little code as possible to make your test fail.
+1. Green: Write as little code as possible to make your test pass.
+1. Refactor: Optimise your code for readability without changing its behaviour.
+
+First, create a test class that calls the intended target class:
+
+``` java
+public class FibonacciTest {
+    
+    Fibonacci target = new Fibonacci();
+
+}
+```
+
+Then create that target class.
+
+``` java
+public class Fibonacci {
+}
+```
+
+Second, add a simple method to the test class.
+
+``` java
+public class FibonacciTest {
+
+    public void shouldBe() {
+        Fibonacci target = new Fibonacci();
+        target.get(0);
```

Comment:
Let's assign this to an `int` variable too, so that we can justify making (or let IntelilJ make) a method that returns an `int`. Otherwise the simplest bit of production code we could write would be a `void` method.
Similarly, let the method accept an `int` parameter at this stage. This is missing from some of the stages below.
Writing as little test code as possible is good, but let's write it 'in the right direction' if you know what I mean.